Add bevy

guimcaballero opened the pull request:
Adds link to the [Bevy Engine](https://bevyengine.org/) official discord server.

mhxion reviewed (changes requested):
Hey @guimcaballero thanks for contributing, just finished reviewing your server, it's a great game engine and a community. Can you please replace it on alphabetical order, right after `Armory`, and right before `Construct Community` (line `1449`)? Everything else is good to be merged.

guimcaballero commented:
Oops, didn't realize they were sorted! Fixed now. 

Also, I've seen that some have a `<br />` after them, so I've added one too. Should I remove it?

mhxion reviewed (approved):
Thanks. Those `</ br>` are there to fix overlapping issues that `align="left"` creates. No need to modify the `</ br>` for now though.
